pipeline/tasks/python_grpc_tasks.py

```diff
@@ -56,9 +56,12 @@ def execute(self, src_proto_path, import_proto_path, common_protos_yaml):
         with open(common_protos_yaml) as common_protos_file:
             common_protos_data = yaml.load(common_protos_file)
 
-        # Treat google.protobuf as a common proto package, even though it is
-        # not included in the common-protos we generate.
-        common_protos = ['google.protobuf']
+        # Treat google.protobuf, google.iam as a common proto package, even
+        # though they are not included in the common-protos we generate.
+        #
+        # TODO (geigerj): remove 'google.iam' when it is included in the common
+        # protos package.
+        common_protos = ['google.protobuf', 'google.iam']
```

Comment:
Shouldn't this include everything in https://github.com/geigerj/toolkit/blob/59e0385f4dd580610a130cd57116ed6dc65ca0a9/src/main/java/com/google/api/codegen/py/PythonImportHandler.java#L42-L49?